docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: "3.9"
+
+services:
+  mongodb:
+    image: mongo
+    ports:
+      - 27017:27017
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=mongo
+      - MONGO_INITDB_ROOT_PASSWORD=mongo
+      - MONGO_INITDB_DATABASE=animavita
```

Would be nice to also persist the data in a volume otherwise it would (correct if I am wrong) be deleted every time you destroy the container

```suggestion
version: "3.9"

services:
  mongodb:
    image: mongo
    ports:
      - 27017:27017
    environment:
      - MONGO_INITDB_ROOT_USERNAME=mongo
      - MONGO_INITDB_ROOT_PASSWORD=mongo
      - MONGO_INITDB_DATABASE=animavita
   volumes:
     - mongo:/data/db
volumes:
  mongo:
```

---

Definitely, thanks!